### PR TITLE
LocationChangedEventArgs and default value for Heading

### PR DESCRIPTION
**Pull request by tsandeepkumar**
Please update your branch with my changes. I added a new class LocationChangedEventArgs that contains both GeoPosition and Heading. I also set Heading to NaN if the heading has no value.

### CODE REVIEW

**Comment by EddieGarmon** [comment]
Please add tests to verify the NaN change.

**Comment by EddieGarmon** [comment]
Please make the LocationChangedEventArgs a read only class.

File: Source/GraduatedCylinder.Geo Specs/Devices/Gps/Nmea/GPRMCSpec.cs [hunk 1]
```diff
@@ -25,5 +25,18 @@ public class GPRMCSpec
             provideGeoPosition.CurrentLocation.Longitude.Value.ShouldEqualWithinPrecision(longitude, 10);
             provideGeoPosition.CurrentLocation.Longitude.Hemisphere.ShouldEqual(longHemisphere);
         }
+
+        [Theory]
+        [InlineData("$GPRMC,145710.00,A,3605.08678,N,07958.72572,W,0.037,,180716,,,D*6B")]
+        public void EmptyHeading(string nmea) {
```
Comment (EddieGarmon) [inline Source/GraduatedCylinder.Geo Specs/Devices/Gps/Nmea/GPRMCSpec.cs]: Here lets make this test 'ValidateHeading' and pass in 2 parameters, the sentence and the expected heading. Lets test for both specified and missing headings.

**Comment by tsandeepkumar** [comment]
Done

**Comment by EddieGarmon** [comment]
merged and released in 0.60.0

**Pull request closed without merging**